packages/mgt-components/src/styles/shared-styles.scss

```
@@ -29,6 +29,11 @@
   margin: 4px 0;
 }
 
+.error {
+  background-color: rgb(253, 231, 233);
+  padding: 8px 12px 8px 8px;
```

we can use the inline and block short hands to make the rtl switch simpler
```suggestion
  padding-block: 8px;
  padding-inline: 8px 12px;
```

---

You tell me! Good idea, wasn't aware of this!